cypress/plugins/index.js

```diff
@@ -298,7 +298,14 @@ module.exports = async (on, config) => {
             jsonReport = readDataFromFile(filePath + fileName);
           }
           const reportProperties = {};
+          let customReportData;
+          try {
+            customReportData = require('../../node_modules/configModule/cypress/fixtures/objects/customReportData.json');
```

Comment:
Can you confirm if this does not get copied to `FCS/cypress/fixtures/external` when the setup is being done?